### PR TITLE
Renamed Ansible parameter from vendor to sftp_path

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
see also corresponding PR : https://github.com/Linaro/hpc_lab_setup/pull/50

### CODE REVIEW

File: deploy_benchmarks.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@
       with_items: "{{ reports_path.results[0].files }}"
 
     - name: Copy results to fileserver
-      shell: (echo "put -r results/{{ sftp_dirname }}" && echo "exit") | sftp -o ForwardAgent=yes -o ConnectTimeout=60 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no  {{sftp_user}}@{{ sftp_server_ip }}:/{{ vendor }}/benchmark
+      shell: (echo "put -r results/{{ sftp_dirname }}" && echo "exit") | sftp -o ForwardAgent=yes -o ConnectTimeout=60 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no  {{sftp_user}}@{{ sftp_server_ip }}:/{{ sftp_path }}/benchmark
```
Comment (rengolin) [inline deploy_benchmarks.yml]: sftp_path must include "/benchmark", too

Comment (rengolin) [reply deploy_benchmarks.yml]: Also, we need to make this conditional on SFTP being used at all. SFTP should NOT be a requirement.

Comment (BaptisteGerondeau) [reply deploy_benchmarks.yml]: SFTP as an option is necessary, I agree, but outside the scope of this PR.

File: deploy_benchmarks.yml [hunk 2]
```diff
@@ -85,7 +85,7 @@
       with_items: "{{ reports_path.results[0].files }}"
 
     - name: Copy results to fileserver
-      shell: (echo "put -r results/{{ sftp_dirname }}" && echo "exit") | sftp -o ForwardAgent=yes -o ConnectTimeout=60 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no  {{sftp_user}}@{{ sftp_server_ip }}:/{{ vendor }}/benchmark
+      shell: (echo "put -r results/{{ sftp_dirname }}" && echo "exit") | sftp -o ForwardAgent=yes -o ConnectTimeout=60 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no  {{sftp_user}}@{{ sftp_server_ip }}:/{{ sftp_path }}
```
Comment (BaptisteGerondeau) [inline deploy_benchmarks.yml]: sftp_path now contains the absolute path to the place to put the results directory in
It is fully populated in the benchmark job (https://github.com/Linaro/hpc_lab_setup/pull/50)

**Comment by rengolin** [comment]
Fair enough.

**Pull request merged**